source/IntegrationTests/Application/Transactions/WholesaleCalculations/MonthlyAmountPerChargeResultProducedV1Tests.cs

```diff
@@ -95,13 +95,13 @@ public async Task MonthlyAmountPerChargeResultProducedV1Processor_creates_outgoi
     }
 
     [Fact]
-    public async Task MonthlyAmountPerChargeResultProducedV1Processor_creates_outgoing_message_to_energy_supplier_and_energinet_as_charge_owner()
+    public async Task MonthlyAmountPerChargeResultProducedV1Processor_creates_outgoing_message_to_energy_supplier_and_system_operator_as_charge_owner()
     {
         var monthlyPerChargeEvent = _monthlyPerChargeEventBuilder
-            .WithChargeOwner(DataHubDetails.DataHubActorNumber.Value)
+            .WithChargeOwner(DataHubDetails.SystemOperatorActorNumber.Value)
             .Build();
         await HandleIntegrationEventAsync(monthlyPerChargeEvent);
-        await AssertOutgoingMessageAsync(ActorRole.EnergySupplier);
+        await AssertOutgoingMessageAsync(ActorRole.SystemOperator);
```

Comment:
```suggestion
        await AssertOutgoingMessageAsync(ActorRole.SystemOperator);
```
delete?
Dublicated lines (105)